package.json

```diff
@@ -36,26 +35,18 @@
     "babel-cli": "^6.3.15",
     "babel-core": "^6.3.15",
     "babel-eslint": "^7.1.1",
-    "babel-jest": "^20.0.3",
     "babel-loader": "^6.2.0",
     "babel-preset-es2015": "^6.3.13",
     "babel-preset-react": "^6.3.13",
     "babel-preset-stage-1": "^6.3.13",
-    "bluebird": "^3.0.5",
-    "chai": "^3.2.0",
-    "cssnext-loader": "^1.0.1",
     "eslint": "^3.17.1",
     "eslint-config-littlebits": "^0.6.2",
-    "jest": "^19.0.2",
-    "jest-cli": "^19.0.2",
     "prettier": "^1.7.3",
     "ramda": "^0.18.0",
-    "react": "^15.4.2",
-    "react-dom": "^15.4.2",
+    "react": "16",
```

Comment:
probably would be better to not use exact v16 and instead use `^16.0.0`

Reply:
Ah yeah, not used to `yarn upgrade@...` yet. Shouldn't we make react deps peer deps too?

Reply:
Yeah https://github.com/reactjs/react-modal/blob/master/package.json#L63-L66

Reply:
But meh this is still an issue https://github.com/yarnpkg/yarn/issues/2479 so whatever will leave peer deps definition for now.